examples/transportation/docusaurus/access-restriction-04-axle-limit.yaml

```diff
@@ -0,0 +1,21 @@
+---
+id: access-restrictions-segment-axle-limit
+type: Feature
+geometry:
+  type: LineString
+  coordinates:
+    - [0, 0]
+    - [0, 1]
+properties:
+  theme: transportation
+  type: segment
+  version: 0
+  updateTime: "2023-06-20T17:19:00-06:00"
+  subType: road
+  road:
+    restrictions:
+      access:
+        - denied:
+            when:
+              mode: [hgv]
+              vehicle: { axleCount: { isAtLeast: 5 } }
```

Comment:
nit: would be more readable if we model it use isMoreThan instead of isAtLeast

Reply:
I'm gonna leave it as-is, just to drag this PR kicking and screaming over the finish line, but I have absolutely no issue with anyone going in and changing it. It's not so much that I think it's right, as that it's not wrong enough to change today!